pkg/osquery/tables/dataflattentable/exec_and_parse.go

```diff
@@ -0,0 +1,101 @@
+package dataflattentable
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+type flattenBytesInt interface {
```

Comment:
Apologies if I missed this one. I'm opposed to using the `Int` convention to name interfaces. I automatically assume it has something to do with integer. I think this should be called `bytesFlattener`.